core/src/types.ts

```diff
@@ -269,22 +269,27 @@ export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, Com
 
 type CollateralType = CollateralConfig['title'];
 
+export declare interface CollateralLiquidationLimit {
+    currentCollateralDebtDai: BigNumber;
+    maximumCollateralDebtDai: BigNumber;
+}
 export declare interface LiquidationLimits {
     maximumProtocolDebtDai: BigNumber;
-    maximumCollateralDebtDai: BigNumber;
     currentProtocolDebtDai: BigNumber;
-    currentCollateralDebtDai: BigNumber;
+    collateralDebts: Record<CollateralType, CollateralLiquidationLimit>
 }
 
-export declare interface VaultCollateralParameters extends LiquidationLimits {
+export declare interface VaultCollateralParameters {
     stabilityFeeRate: BigNumber;
-    maxDebtPerCollateralUnit: BigNumber;
+    minUintPrice: BigNumber;
 }
 
-export declare interface VaultBase {
+export declare interface Vault {
```

Comment:
This type doesn't contain enough to announce it being `liquidatable`. Therefore I will still call this type `VaultBase`

Reply:
Meaning [the suggestion](https://github.com/sidestream-tech/unified-auctions-ui/pull/446/files#r963578099) is not fully implemented yet as there is no `Vault` type yet